portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -58,9 +61,10 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     PreparedQuery results = datastore.prepare(query);
 
-    String userChoice = request.getParameter("num");
+    String numChoice = request.getParameter("num");
+    String languageChoice = request.getParameter("language");
     int maxComments;
-    maxComments = Integer.parseInt(userChoice);
+    maxComments = Integer.parseInt(numChoice);
     int i =0;
     for (Entity entity : results.asIterable()) {
```

Comment:
If you add a maven dependency on [Guava](https://mvnrepository.com/artifact/com.google.guava/guava), you could use [`Iterables.limit`](https://guava.dev/releases/21.0/api/docs/com/google/common/collect/Iterables.html#limit-java.lang.Iterable-int-) rather than manually counting iterations.

Reply:
I am a little confused on how to do this. I added the maven dependency and Iterables.limit(results.asIterable(), maxComments), but it is saying it cant find symbol Iterables.

Reply:
done!